test/UnitTest/Microsoft.AspNet.OData.Test.Shared/Formatter/Deserialization/ODataDeserializationTestsCommon.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License.  See License.txt in the project root for license information.
+
+using System.Net.Http;
+using System.Net.Http.Headers;
+using Microsoft.OData;
+using Microsoft.OData.Edm;
+
+namespace Microsoft.AspNet.OData.Test.Formatter.Deserialization
+{
+    internal class ODataDeserializationTestsCommon
+    {
+
```

Comment:
You don't need to create a new file to hold these two methods, you can leave them in the ODataEnumDescrializerTests.cs and make them public static method.

Reply:
That would have been the simpler thing to do but referencing ODataEnumDeserializer in ODataPrimitiveDeserializer code did not seem the right thing to do.